src/components/SearchResults/Facets/FacetGroup.js

```diff
@@ -53,7 +54,7 @@ const FacetGroup = ({ facetOption: { name, id, options }, isNested = false }) =>
         glyph: 'ChevronUp',
         text: 'Show less',
       };
-
+  const siblingsSelected = numSelectedChildren > 1 ? true : false;
```

Comment:
The ternary here might be redundant since we're just returning a boolean equivalent to the condition. It might make sense to simplify

```suggestion
  const siblingsSelected = numSelectedChildren > 1;
```

Reply:
New staging link: https://docs-mongodb-org-stg.s3.us-east-2.amazonaws.com/master/landing/maya/DOP-4077/index.html (will also update above)